cmd/brew-bootstrap-nodenv-node

```diff
@@ -11,6 +11,35 @@ fi
 warn() { echo "$@" >&2; }
 abort() { EXPECTED_EXIT="1"; warn "$@"; exit 1; }
 
+abort_for_sh() {
+  abort 'Error: add `eval "$(nodenv init -)"` to the end of your .bash_profile!'
+}
+
+abort_for_zsh() {
+  abort 'Error: add `eval "$(nodenv init -)"` to the end of your .zshrc!'
+}
+
+abort_for_fish() {
+  abort 'Error: add `status --is-interactive; and . (nodenv init -|psub)` to the end of your .config/fish/config.fish!'
+}
```

Comment:
I don't know if we'll ever actually use them again, but I feel like I like them as separate functions. Glad to fold it in if you have a strong opinion though!